lighthouse-core/report/html/renderer/util.js

```diff
@@ -484,8 +484,8 @@ Util.UIStrings = {
   /** Label of value shown in the summary of critical request chains. Refers to the total amount of time (milliseconds) of the longest critical path chain/sequence of network requests. Example value: 2310 ms */
   crcLongestDurationLabel: 'Maximum critical path latency:',
 
-  /** Explanation shown to users below performance results to inform them that the test was done with a 3G network connection and to warn them that the numbers they see will likely change slightly the next time they run Lighthouse. 'Lighthouse' becomes link text to additional documentation. */
-  lsPerformanceCategoryDescription: '[Lighthouse](https://developers.google.com/web/tools/lighthouse/) analysis of the current page on emulated 3G. Values are estimated and may vary.',
+  /** Explanation shown to users below performance results to inform them that the test was done with a 4G network connection and to warn them that the numbers they see will likely change slightly the next time they run Lighthouse. 'Lighthouse' becomes link text to additional documentation. */
+  lsPerformanceCategoryDescription: '[Lighthouse](https://developers.google.com/web/tools/lighthouse/) analysis of the current page on emulated 4G. Values are estimated and may vary.',
```

Comment:
this ends up prominent in the report, so is it worth adding the extra "slow" in front of "4G" for this string?

Reply:
I took the approach that if it wasn't worth qualifying `fast` before than why qualify `slow` now, but I see both sides.

Reply:
yeah, makes sense. I was thinking in terms of easing in users who will still be thinking in terms of "3G is what I should be testing", rather than specific speeds/network characteristics

Reply:
yeah that's a good point, I like the more general `emulated mobile network` here too actually :)

Reply:
💥 nice